app/utils/user_context.py

```
@@ -96,3 +96,41 @@ def clear_global_user_context():
+            "trading_config.auto_trading_enabled": True
+        })
+        
+        user_ids = [user.get("user_id") for user in active_users if user.get("user_id")]
+        
+        # 활성 사용자가 없으면 기본 사용자만 반환
+        if not user_ids:
+            logger.info("활성 사용자가 없어 기본 사용자만 반환합니다.")
+            return [get_default_user_id()]
+        
+        logger.info(f"활성 사용자 {len(user_ids)}명 조회: {user_ids}")
+        return user_ids
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)